src/wp-includes/rest-api/endpoints/class-wp-rest-templates-controller.php

```diff
@@ -507,11 +529,19 @@ protected function get_available_actions() {
 	 */
 	public function get_collection_params() {
 		return array(
-			'context' => $this->get_context_param(),
```

Comment:
Suggestion: Add an @since comment to the method's DocBlock to reflect the additions:

```
	 * @since 5.8.0
	 * @since 5.9.0 Added `'area'` and `'post_type'`.
```